README.md

```diff
@@ -1,150 +1,201 @@
-# Bookstore
+<a name="readme-top"></a>
 
-> Booktore is an application whivh helphs with books storage. You can display a list of books, add a book or remove a selected book.
+<!--
+HOW TO USE:
+This is an example of how you may give instructions on setting up your project locally.
 
-## Built With
+Modify this file to match your project and remove sections that don't apply.
 
-- React.js
-- Git & GitHub
+REQUIRED SECTIONS:
+- Table of Contents
+- About the Project
+  - Built With
+  - Live Demo
+- Getting Started
+- Authors
+- Future Features
+- Contributing
+- Show your support
+- Acknowledgements
+- License
 
+After you're finished please remove all the comments and instructions!
+-->
 
-## Get Started
+<div align="center">
 
-To get the content of this project locally you need to run the following commands in your terminal.
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [ ] [OPTIONAL] Kindly add this image to your repo to appear in your README file.

Reply:
Hello @gemmen29 !!!
Thanks for the review. I kept the image in the README file. I didn't remove it.